framework-docs/modules/ROOT/pages/integration/cache/annotations.adoc

```diff
@@ -67,7 +67,7 @@ To provide a different default key generator, you need to implement the
 The default key generation strategy changed with the release of Spring 4.0. Earlier
 versions of Spring used a key generation strategy that, for multiple key parameters,
 considered only the `hashCode()` of parameters and not `equals()`. This could cause
-unexpected key collisions (see {issues-old}/SPR-10237[SPR-10237]
+unexpected key collisions (see https://jira.spring.io/browse/SPR-10237[SPR-10237]
```

Comment:
This remaining link somehow didn't show up in my local searches.

It can be replaced with the [GitHub issue](https://github.com/spring-projects/spring-framework/issues/14870).

```asciidoc
{spring-framework-issues}/14870[gh-14870]
```

Reply:
that's my bad, I somehow reintroduced the wrong link / missed the fact that there were 2 jira links in the doc 😮‍💨  will fix right away